fe2-android/app/src/main/java/com/github/dedis/student20_pop/detail/LaoDetailViewModel.java

```diff
@@ -141,6 +144,61 @@ protected void onCleared() {
         disposables.dispose();
     }
 
+    /**
+     * Sends a ElectionCastVotes message .
+     *
+     * <p>Publish a GeneralMessage containing ElectionCastVotes data.
+     *
+     * @param castVotes the corresponding votes for that election
```

Comment:
Instead of passing the cast vote as a parameter you can create it inside, by getting the lao id from the lao channel and passing the list of ElectionVote and the election id as parameters. Similar to `createNewElection`, `createNewRollCall`, etc.